go/adapter.go

```diff
@@ -83,3 +83,25 @@ func (b *AdapterBase) Stop(wait bool) {
 		b.eventBucket.Wait()
 	}
 }
+
+// Definition of how to filter our Spans to reduce noise
+type SpanFilter struct {
+	MinDuration time.Duration
+}
+
+// Specify options to change what or how the adapter receives ObserveEvents
+type Options struct {
+	SpanFilter        *SpanFilter
+	ChannelBufferSize int
+}
+
+// Create a default configuration
+func NewDefaultOptions() *Options {
+	minDuration, _ := time.ParseDuration("50us")
```

Comment:
Should we do a smaller time frame than this? 50 microseconds actually seems like a lot. A lot of my rust programs in their entirety, when the input is small, finish in 100-200us.

Reply:
I'm comfortable with a smaller timeframe.. What do you think?  20us, 10us?

Reply:
I went with 20us for now